Rakefile

```diff
@@ -17,7 +17,7 @@ task :default => [:test]
 
 desc 'Run RSpec'
 RSpec::Core::RakeTask.new(:test) do |t|
-  t.pattern = 'spec/{unit}/**/*.rb'
+  t.pattern = 'spec/{classes,defines,unit,functions,hosts,integration}/**/*_spec.rb'
```

Comment:
You need to do a `rake spec_prep` prior to `rake test` in a clean repo.  Not sure how this is expressed in the rakefile or doco.